### PR TITLE
Revert retry conditions

### PR DESCRIPTION
**Pull request by danielmkn**

### CODE REVIEW

File: pkg/xray/resource_xray_watch_test.go [hunk 1]
```diff
@@ -99,18 +99,18 @@ func TestAccWatch_singleRepository(t *testing.T) {
 	tempStruct["watch_name"] = fmt.Sprintf("xray-watch-%d", randomInt())
 	tempStruct["policy_name_0"] = fmt.Sprintf("xray-policy-%d", randomInt())
 	tempStruct["watch_type"] = "repository"
-	repo0 := "libs-release-local"
-	repo1 := "libs-release-local-1"
+	tempStruct["repo0"] = fmt.Sprintf("libs-release-local-0-%d", randomInt())
```
Comment (alexhung) [inline pkg/xray/resource_xray_watch_test.go:102]: Why do we need to use a struct?

Comment (danielmkn) [reply pkg/xray/resource_xray_watch_test.go:102]: it's not a struct, it's a map of strings. Just the name, which I borrowed in the Artifactory provider.

Comment (alexhung) [reply pkg/xray/resource_xray_watch_test.go:102]: Ok, why are we using a map of strings? Why is this necessary?

Comment (danielmkn) [reply pkg/xray/resource_xray_watch_test.go:102]: I'm using the template to construct HCL. Template uses the map to fill in values into the template

Comment (alexhung) [reply pkg/xray/resource_xray_watch_test.go:102]: Ok then please rename it with more descriptive name. `tempStruct` is just wrong. It isn't temporary, or a struct. It tells me nothing about what it is for.

**Review by alexhung — CHANGES_REQUESTED** [review]

File: pkg/xray/resource_xray_watch_test.go (hunk 1 above)
Comment (danielmkn) [reply pkg/xray/resource_xray_watch_test.go:102]: renamed to testData

**Review by alexhung — APPROVED** [review]

**Pull request merged**